.github/scripts/trymerge.py

```diff
@@ -1291,6 +1291,16 @@ def merge(pr_num: int, repo: GitRepo,
 
     check_for_sev(org, project, force)
 
+    gh_post_pr_comment(org, project, pr.pr_num, explainer.get_merge_message(land_check_commit))
```

Comment:
land_check_commit will always be None here if we move this.

Reply:
+1, also, it makes sense to post this comment _after_ we've done the first round of sanity checks to ensure we'll actually try to merge the pr